src/Components/MovieDetails/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import moviesFromJson from '../../assets/data/movies.json';
+import './style.scss';
+
+function MovieDetails(props) {
+  const [movie, setMovie] = useState({});
+
+  useEffect(() => {
+    Promise
+      .resolve(moviesFromJson)
+      .then(movies => setMovie(movies.find(m => m.id === props.id)));
+  }, [props.id]);
+
+  return (
+    <>
```

Comment:
redundant Fragment :) Please don't forget that you don't need to wrap component in a Fragment if there is only 1 node